repos/fdbt-site/src/interfaces/index.ts

```diff
@@ -734,3 +734,16 @@ export interface Cap {
     durationAmount: string;
     durationUnits: ExpiryUnit;
 }
+
+export interface CapPricePerDistances {
+    distanceFrom: string;
+    distanceTo: string;
+    minimumPrice?: string;
+    maximumPrice?: string;
+    pricePerKm?: string;
```

Comment:
Isnt price per km linked to every row?

I would have thought the data would be stored something like this

export interface CapDistancePricing {
distanceFrom: number; (or string - whichever suits you)
distanceTo: number; (or string - whichever suits you)
pricePerKm: string;
}

export interface DistanceCap {
maximumPrice: string;
minimumPrice: string;
capPricing: CapDistancePricing[]
}